ui/packages/app/src/pages/project/Project.js

```diff
@@ -23,23 +23,39 @@ const Project = () => {
   const { apps } = useContext(ApplicationsContext);
   const { projectId, project } = useContext(CurrentProjectContext);
 
+  const [projectName, setProjectName] = useState("");
+  useEffect(() => {
+    if (project) {
+      setProjectName(project.name);
+    }
+  }, [project]);
+
   const [{ data: entities }, fetchEntities] = useFeastCoreApi(
-    `/feast.core.CoreService/ListEntities`,
-    { method: "POST" },
+    `/feast/api/entities?project=${projectName}`,
```

Comment:
I am wondering whether we should set the endpoint to /entities and append /feast/api to the configuration, similar to other components in MLP.

Reply:
Ah, that's a good catch. I agree with you. Updating it now.